### ISSUE TITLE
REQ001 configures a property by NAME, and CONTEXT.md's identity rule was never extended to properties

### ISSUE THREAD

**Issue by MrBinnacle** [MrBinnacle/workspace_lint#78]
`CONTEXT.md`'s identity rule governs resources: *identity is the stable ID, and a name is a
report-only alias that is never resolved.* The rule-configuration section built for #19 applies it to
a rule's `scope`, which is a resource. It does **not** apply it to `REQ001`'s `property`, and that is
an unmade decision rather than a settled one.

## The exposure

`slice/config.ts` validates `property` as a non-empty string and nothing more:

```jsonc
{ "rule": "REQ001", "scope": { "id": "…" }, "property": "Owner" }
```

A Notion property carries **both** a stable `id` and a mutable `name`. The argument that makes a
resource's identity its ID reaches a property unchanged:

- **The failure is silent and it is in the flattering direction.** Rename `Owner` to `Assignee` in
  Notion and `REQ001` stops matching. It does not report that the configured property vanished; it
  reports over whatever the scope still contains. The operator sees a rule that passes.
- **It is the same mechanism `#19` rejected for roots.** A name that an editor can change decides
  what the scan evaluates.

## Why it was not settled inside #19

Property IDs are opaque short strings — `title`, `%3AUUq` — and the Notion UI does not surface them.
Requiring one may be **unusable** rather than merely strict, which is a different objection from the
one that governs resources: a page ID is obtainable from the page's own URL, and a property ID is not
obtainable from anything a user can see.

That asymmetry is the whole question, and it needs the API checked rather than reasoned about.

## What would settle it

1. **Does `GET /v1/data-sources/{id}` return the property `id` alongside the `name`?** If it does,
   the tool can resolve a configured name to an ID **once, at config-authoring time, writing the ID
   into the file** — which is precisely the escape #19's own Revisit-if already names for the root ID
   requirement, and it never resolves at scan time.
2. **Does a rename preserve the property `id`?** If a rename mints a new ID, an ID is no more stable
   than a name and the whole question dissolves.
3. If neither holds, the answer is a **disclosure**, not a rejection: `REQ001` states in its report
   that its subject is a property name and that a rename is indistinguishable from a deletion.

Both questions are live-API facts. `docs/proof/` is where the answer belongs, not `docs/research/`.

## Blocking edges

**Blocks nothing.** #58 can build `REQ001` against a property name; this decides whether the name is
accompanied by a check, an authoring-time resolution, or a printed caveat. It should be settled
**before** `REQ001`'s report format is fixed, because option 3 changes what the report prints.

## Revisit if

**Notion surfaces property IDs in its UI.** The usability objection is the only thing separating this
from the resource rule, and it disappears the day a user can copy the ID.

**Comment by MrBinnacle** [comment]
Labelled `deferred` — **recorded, not scheduled**. This is the amended standing rule, 2026-08-18: the original refused to FILE a decision ticket until four rules ship, which limited recording where the problem was scheduling. Filing is now free; the limit is on ACTIVE decisions (at most one, while a v0.1 rule is unbuilt). `docs/agents/triage-labels.md` carries the amendment and the reasoning.

The amended rule requires every deferral to name what would make it active. Proposed trigger below — a **proposal from a session that did not re-read this issue's full history**, so correct it if the issue's own record names a better one.

**Revisit if:** a property is renamed in a real workspace and `REQ001` silently changes what it evaluates, or the Notion UI is shown to surface a property ID an operator can copy — this issue's own premise is UNSOURCED on that point.

**New evidence from #58, which does not settle it.** `REQ001` shipped configuring the property by NAME and keeping this question open; the discriminator stores the property as configured, so a later switch to IDs is a config migration rather than a rule rewrite. And the live run observed that `GET /v1/pages/{id}` **does** return a property ID — 3 of 4 pairs carried one — where `docs/research/notion-live-probe.md` Probe 3 observed none over the OAuth connector (`docs/proof/results-58-req001.md` §5.1). That the API returns an ID says nothing about whether a human can obtain one, which is the half this issue turns on.

> *This was generated by AI during triage.*
